src/other/Table.js

```diff
@@ -36,6 +36,7 @@
 
     Table.prototype.publish("quickUpdateOnClick", false, "boolean", "Avoid calling update on row select/deselect", null, { tags: ["Private"] });
     Table.prototype.publish("hideEmptyColumns", false, "boolean", "Hide columns with all empty cells");
+    Table.prototype.publish("hiddenColumnIdxArr", [], "array", "Hide columns by column index");
```

Comment:
How is this different to the already existing `hiddenColumns` PP?

Reply:
It isn't. I overlooked that publish parameter.